tests/index.js

```diff
@@ -829,6 +868,39 @@ describe('Thorn', () => {
                 yield Agent.as('TestUsername').get('not/real/endpoint');
                 expect(server.isDone()).to.be.true;
             });
+
+            it('should reset Fixtures tokens when using the Admin', function*() {
+                let server = nock(process.env.THORN_SERVER_URL)
+                    .post(isTokenReq)
+                    .reply(200, ACCESS)
+                    .post(isBulk)
+                    .reply(200, () => constructBulkResponse({
+                        _module: 'TestModule1',
+                        name: 'DummyRecord1',
+                    }))
+                    .post(isTokenReq)
+                    .reply(200, {
+                        access_token: 'New-Access-Token',
+                        refresh_token: 'New-Refresh-Token',
+                    })
+                    .get(/not\/real\/endpoint/)
+                    .reply(200);
```

Comment:
Remind me to run this with THORN_VERBOSE=1 set to ensure that I've got `nock` completely set up right. I might be missing something here.

Reply:
Just did; it's correct.